### ISSUE TITLE
valid loss

### ISSUE THREAD

**Issue by YukiVerrckt** [manhdung20112000/speech-enhancement#6]
Hello, I trained your model using LibriSpeech Dataset. while the valid loss is rising slowly. Have you encountered the similar problem?
step:200 valid loss:0.0040 ---> step:1300 valid loss: 0.0042 
![image](https://user-images.githubusercontent.com/47894625/155283663-eb9bf3ba-a2ad-402a-aa04-8c087b0c3c44.png)

**Comment by nmd2k** [comment]
Hi, sorry for this (very) late reply. Have you managed to encounter the problem?
Does the shape of `training loss` similar to this, I assume this might be a good sign that your model has been saturated.
From my experiment, the dataset was quite small and I intentionally keep the U-net simple to avoid overfit and hopefully it can generalize the knowledge.

To sum up, I don't think this is something, the model may not be underfitted or overfitted, how about validating this prediction with some real metrics?

*Stay safe*

**Comment by nmd2k** [comment]
If there is nothing more, I will close this issue. Feel free to open if when you need.
